openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -1664,6 +1664,13 @@ for example, by including clear-text session information as a `state` parameter
 it in a `redirect_uri` parameter. A third party may observe such information through browser
 history, etc. and correlate the user's activity using it.
 
+### Wallet Attestation Subject {#walletattestation-sub}
+
+The Wallet Attestation as defined in (#wallet attestation) SHOULD NOT introduce a unique identifier specific to a single client.
```

Comment:
```suggestion
The Wallet Attestation as defined in (#walletattestation) SHOULD NOT introduce a unique identifier specific to a single client.
```

Reply:
shouldn't this be "MUST NOT"?

Reply:
Good question and for most systems I would agree. There might be ecosystems with other privacy properties and I wasn't sure if we should disallow that in the core spec -> I went with SHOULD, but I am happy to change

Reply:
but wallet attestation format is defined in VCI, right? so i think it should be MUST NOT. i might be short sighted but what other privacy property can there be?

Reply:
I am happy to continue the discussion in https://github.com/openid/oid4vc-haip/issues/171